Gems/ScriptAutomation/Code/Source/Utils/Utils.cpp

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include <Utils/Utils.h>
+
+#include <AtomCore/Instance/InstanceDatabase.h>
+#include <Atom/RHI/RHISystemInterface.h>
+#include <Atom/RPI.Public/RenderPipeline.h>
+#include <Atom/RPI.Public/RPISystemInterface.h>
+#include <Atom/RPI.Public/Scene.h>
+#include <Atom/RPI.Public/Image/ImageSystemInterface.h>
+#include <Atom/RPI.Reflect/Asset/AssetUtils.h>
+#include <Atom/RPI.Reflect/Image/StreamingImageAssetCreator.h>
+#include <Atom/RPI.Reflect/Image/ImageMipChainAssetCreator.h>
+#include <Atom/Utils/DdsFile.h>
+
+#include <AzFramework/CommandLine/CommandLine.h>
```

Comment:
This include isn't used.
(I was originally going to state that we prefer to get the CommandLine and StringFunc include through `<AzCore/Settings/CommandLine.h>` and <AzCore/StringFunc/StringFunc.h>`